emulators/xhyve/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        mist64 xhyve 0.2.0 v
-revision            2
+github.setup        mist64 xhyve 1f1dbe3 
+revision            0
```

Comment:
I don't control the upstream xhyve repo, so I can't provide version tags.

I forgot that Macports needs versions to be increasing; subsequent hashes will appear sometimes to Macports as downgrades.

I'll open an issue with @mist64 to tag a new version, but in the meantime, will the automatic git adapter parse the hash out of the version string like 1f1dbe3 from 0.2.1-git-1f1dbe3 ??